### ISSUE TITLE
Je ne connais pas les pouces de mon téléphone

### ISSUE THREAD

**Issue by DavidBruant** [betagouv/Ecolab#142]
> Quel téléphone avez-vous ?

J'ai répondu un peu au pif parce qu'entre "moins de 4,5 pouces" et "moins de 5 pouces", je ne sais pas faire la différence

**Comment by laem** [comment]
Oui, bonne remarque. D'autant plus que depuis déjà plusieurs années (probablement plusieurs générations de téléphones), on ne peut plus acheter de téléphone < 5.5 pouces, petit iPhone SE excepté. 
Je vais enlever cette question.
